src/functionality.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-use-before-define */
+const taskList = document.getElementById('task-list');
+
+let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+export function addTask(description) {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length,
```

Comment:
* As stated in the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=By%20default%20new,set%20to%201)..) `if you're adding a 5th task to the list, the index of that task should equal to 5`  and `if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1`  **HINT** I think setting index to = 1 and increment on the addition of every task would work as setting it to `task.length` would always start from 0 since arrays starts from index 0

Reply:
issue fixed